clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -270,6 +279,17 @@ class DockerRegistryClient {
    * don't want you to download their whole catalog (it's potentially a lot of data).
    */
   public DockerRegistryCatalog getCatalog(String path = null) {
+    if(catalogFile) {
+      log.info("Using catalog list at $catalogFile")
+      try {
+        def userDefinedCatalog = new BufferedReader(new FileReader(catalogFile)).getText()
+        return (DockerRegistryCatalog) new Gson().fromJson(userDefinedCatalog, DockerRegistryCatalog.class)
+      } catch(Exception e) {
+        log.warn("Error encountered reading catalogFile" + e.getMessage())
+        return new DockerRegistryCatalog(repositories: [])
```

Comment:
It might be best to fail to add the account here, this way the logs have a big stack trace with the exception in it rather than a single warning line